### PR TITLE
Add docs sidebar

### PR DESCRIPTION
**Pull request by supertopher**
## What
Adds sidebar with the same layout as the old sidebar

## How
Wrote JSON to match the format provided from the old docs

### CODE REVIEW

**Comment by supertopher** [comment]
Sidebar.js is a large part of this commit and doesn't show by default

File: docs/readme.md [hunk 1]
```diff
@@ -1,15 +1,70 @@
-# docs (by Docusaurus)
+# Introduction
```
Comment (supertopher) [inline docs/readme.md:1]: This is a copy of `/README.md` Docusaurus doesn't like doing above it's root directory.  I tried a symlink and the link appears to fail docusaurus's build steps so I just copied to move past the problem as a blocker

**Comment by supertopher** [comment]
Screen Shot 2022-04-07 at 1 30 59 PM

**Review by Amruta-Ranade — APPROVED** [review]
woooo!!

**Comment by supertopher** [comment]
pushing to update docs

File: docs/readme.md (hunk 1 above)
Comment (davinchia) [reply docs/readme.md:1]: Does this mean we have a duplicate? If we can't figure this out, we could change the workflow to copy the top level readme in this directory as part of the workflow.

@Amruta-Ranade do you have thoughts on the repo readme vs the docs.airbyte.io readme? We've kept it the same for continuity. I want to continue doing so and was wondering if you have other plans.

**Comment by davinchia** [comment]
Nicely done. Is this now live on doc.airbyte.io?

File: docs/readme.md (hunk 1 above)
Comment (Amruta-Ranade) [reply docs/readme.md:1]: I do have plans to edit the landing page text. The repo readme is perfect for the OSS audience but I'd like the docs readme to be more user-centric. Keeping them separate makes sense to me.

**Pull request merged**